### DISCUSSION TITLE
breakpad windows port

### DISCUSSION THREAD

**Discussion by Nemirtingas** [microsoft/vcpkg#33619] [Q&A] [answered]
Hi since some version, breakpad depends on atlmfc pseudo port. Any reasons why ? I never needed to install atlmfc to have breakpad compile and work on my build environments.

**Comment by autoantwort** [comment]
From the referenced issue in the PR: https://github.com/microsoft/vcpkg/issues/29531#issuecomment-1423215632

**Reply by dg0yt** [reply]
FTR there is some discussion about ATL without MFC here: https://github.com/microsoft/vcpkg/pull/31604#issuecomment-1634105374.
But there is no action yet.

**Reply by Nemirtingas** [reply, marked as answer]
I will just nullify this port atlmfc port then, thanks.

**Reply by Nemirtingas** [reply]
Ah, will do later I guess 🤣 
```
<html><head>
<title>SourceForge</title>
<!-- <script src="/js/jquery.com/jquery-1.11.0.min.js"></script> -->
<script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
<script src="https://sourceforge.net/js/mirrors.js"></script>
<script src="/js/sf.js"></script>
<script>
var DR_loc = DR_parse_hash_url();
if (DR_loc) {
    DR_sf_main(DR_loc);
} else {
    window.location.href = 'https://sourceforge.net/home.html';
}
</script>
<meta name="description" content="Free, secure and fast downloads from the largest Open Source applications and software directory - SourceForge.net"/>
</head><body>
<noscript>
We're sorry -- the Sourceforge site is currently in Disaster Recovery mode.  Please check back later.
</noscript>
</body></html>
```

**Reply by Nemirtingas** [reply]
Well ended up with my own github repo, I took the archive from my vcpkg downloads directory as sourceforge seems still down for me :(
https://github.com/Nemirtingas/libdisasm

**Reply by Nemirtingas** [reply]
I pushed a commit on my branch : https://github.com/Nemirtingas/vcpkg/commit/dd6e248a48c02d50b668c80521f3912ca91f1e3e
Is splitting ATL and MFC interesting for vcpkg ?
